docs/example.md

```diff
@@ -0,0 +1,42 @@
+[*Example of usage*](https://tinyurl.com/2aj9lkwf):
+```js
+import 'core-js/actual'; // <- at the top of your entry point
+
+Array.from(new Set([1, 2, 3, 2, 1]));          // => [1, 2, 3]
+[1, 2, 3, 4, 5].group(it => it % 2);           // => { 1: [1, 3, 5], 0: [2, 4] }
+Promise.resolve(42).then(x => console.log(x)); // => 42
+structuredClone(new Set([1, 2, 3]));           // => new Set([1, 2, 3])
+queueMicrotask(() => console.log('called as microtask'));
+```
+
+*You can load only required features*:
+```js
+import 'core-js/actual/array/from';       // <- at the top of your entry point
+import 'core-js/actual/array/group';      // <- at the top of your entry point
+import 'core-js/actual/set';              // <- at the top of your entry point
+import 'core-js/actual/promise';          // <- at the top of your entry point
+import 'core-js/actual/structured-clone'; // <- at the top of your entry point
+import 'core-js/actual/queue-microtask';  // <- at the top of your entry point
+
+Array.from(new Set([1, 2, 3, 2, 1]));          // => [1, 2, 3]
+[1, 2, 3, 4, 5].group(it => it % 2);           // => { 1: [1, 3, 5], 0: [2, 4] }
+Promise.resolve(42).then(x => console.log(x)); // => 42
+structuredClone(new Set([1, 2, 3]));           // => new Set([1, 2, 3])
+queueMicrotask(() => console.log('called as microtask'));
+```
+
+*Or use it without global namespace pollution*:
+```js
+import from from 'core-js-pure/actual/array/from';
+import group from 'core-js-pure/actual/array/group';
+import Set from 'core-js-pure/actual/set';
+import Promise from 'core-js-pure/actual/promise';
+import structuredClone from 'core-js-pure/actual/structured-clone';
+import queueMicrotask from 'core-js-pure/actual/queue-microtask';
+
+from(new Set([1, 2, 3, 2, 1]));                // => [1, 2, 3]
+group([1, 2, 3, 4, 5], it => it % 2);          // => { 1: [1, 3, 5], 0: [2, 4] }
+Promise.resolve(42).then(x => console.log(x)); // => 42
+structuredClone(new Set([1, 2, 3]));           // => new Set([1, 2, 3])
+queueMicrotask(() => console.log('called as microtask'));
+```
```

Comment:
It's better to leave it on the main README.

Reply:
I agree that it should be on the root-main README for easy/fast access, but it seems kinda big. What if we split the examples in 2 parts? (one in root, the other in docs)

Reply:
I'm going to put it back at root. Maybe in the future (if more examples are added) it would be good to consider splitting it